lib/util/github/graphql/cache-adapters/memory-cache-adapter.spec.ts

```diff
@@ -0,0 +1,178 @@
+import { DateTime } from 'luxon';
+import * as memCache from '../../../cache/memory';
+import { clone } from '../../../clone';
+import type { GithubDatasourceItem, GithubGraphqlCacheRecord } from '../types';
+import { GithubGraphqlMemoryCacheAdapter } from './memory-cache-adapter';
+
+const isoTs = (t: string) => DateTime.fromJSDate(new Date(t)).toISO();
+
+const mockTime = (input: string): void => {
+  jest.spyOn(DateTime, 'now').mockReturnValue(DateTime.fromISO(isoTs(input)));
```

Comment:
don't spy, luxon has a special option to override now. search our tests 😉


https://moment.github.io/luxon/api-docs/index.html#settingsnow